docs/notebooks/nmodl-linear-solver.ipynb

```diff
@@ -39,7 +32,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.9.7"
```

Comment:
FYI with
```
jupyter nbconvert --ClearOutputPreprocessor.enabled=True --ClearMetadataPreprocessor.enabled=True --clear-output --inplace docs/notebooks/nmodl-linear-solver.ipynb
```
you can clear all these python and vscode related details from jupyter notebooks